include/jemalloc/internal/tcache_structs.h

```diff
@@ -55,6 +55,8 @@ struct tcache_slow_s {
 
 struct tcache_s {
 	tcache_slow_t	*tcache_slow;
+	unsigned	tcache_nhbins;
+	size_t		tcache_max;
```

Comment:
A request for the next PR: move the pair to tcache_slow, because tcache_t is part of TCACHE_FAST
https://github.com/jemalloc/jemalloc/blob/dev/include/jemalloc/internal/tsd_internals.h#L131

which is accessed directly on the alloc / free fastpath. While these are not required to be there. This is to make sure the metadata needed on the fast path are packed as tight as possible, to reduce the # of cachelines touched.

Assuming we have done the abstraction properly, moving them to tcache_slow should be a small change.